app/cdap/components/common/InlayDrawerWidget/Menu.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Button from '@material-ui/core/Button';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import { getTestIdString } from 'components/common/InlayDrawerWidget/index';
+import { PREFIX } from 'components/common/InlayDrawerWidget/InlayDrawerWidget.stories';
+import T from 'i18n-react';
+import React, { KeyboardEvent, MouseEvent, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+export interface IActionsOptions {
```

Comment:
This needs a better name if its going to be exported

Reply:
renamed interface to IMenuItem